moveit_core/robot_trajectory/include/moveit/robot_trajectory/robot_trajectory.h

```diff
@@ -40,10 +40,17 @@
 #include <moveit/macros/class_forward.h>
 #include <moveit/macros/deprecation.h>
 #include <moveit/robot_state/robot_state.h>
-#include <moveit_msgs/RobotTrajectory.h>
-#include <moveit_msgs/RobotState.h>
+#include <moveit_msgs/msg/robot_trajectory.hpp>
+#include <moveit_msgs/msg/robot_state.hpp>
```

Comment:
an aside: i still really think changes like this would be so much easier to review if done in bulk using find-replace. there's nothing debatable about this, and it would make these PRs quicker to review and merge.